### PR TITLE
admin/update-license-footer-text

### LINKED ISSUE #108: Update footer text to reference creative commons license of generated content
The footer license text is generally accurate. The only addition should be that the "tools and technology of CDP are licensed under MIT, while the content is licensed under CC4" or something similar.

### PR DESCRIPTION
**Pull request by JacksonMaxfield**
### Link to Relevant Issue

This pull request resolves #108 

### Description of Changes

_Include a description of the proposed changes._

Update footer text and related strings to reference the different between our tools and the produced content. Content licensing is weird and if someone uses a transcript of ours or something we should get cited.

### Link to Forked Storybook Site

_If component changes (especially visual changes) are contained in this PR, we ask that you provide a link to a publicly viewable version of the Storybook docs site, so PR reviewers can see your changes without having to install and view your code locally._

image

_Please see `CONTRIBUTING.md` for directions on how this can be done._

### CODE REVIEW

File: src/components/Layout/Footer/Footer.tsx [hunk 1]
```diff
@@ -68,12 +68,36 @@ const Footer: FC<FooterProps> = ({ footerLinksSections }: FooterProps) => {
...
+              <a rel="external" href="https://protocol.mozilla.org/">
+                Mozilla Protocol
+              </a>
+              .
+              <br />
+              {strings.artwork_provided_by_prefix}
+              <a rel="external" href="https://undraw.co/">
+                unDraw
+              </a>
+              .
```
Comment (tohuynh) [inline src/components/Layout/Footer/Footer.tsx:100]: This is not a big deal.

I would put in each license item in its own HTML tag (maybe `<li>` inside an `<ul>`)

I would also put the crediting of Mozilla protocol and undraw outside `<p className="mzp-c-footer-license">` but inside `<div className="mzp-c-footer-legal">`.

**Review by tohuynh — APPROVED** [review]
Did you want the links to open in a new tab? If yes, then add a target="_blank" to the anchor elements.

File: src/components/Layout/Footer/Footer.tsx (hunk 1 above)
Comment (JacksonMaxfield) [reply src/components/Layout/Footer/Footer.tsx:100]: Got it. Will do later tonight!

Comment (tohuynh) [reply src/components/Layout/Footer/Footer.tsx:100]: Scratch that. Just address the second comment only.

Comment (JacksonMaxfield) [reply src/components/Layout/Footer/Footer.tsx]: Resolved as simply just moving each section of content, the copyright, the licenses, and the citing other tools into individual p tags.

**Review by hawkticehurst — APPROVED** [review]
Looks great!

**Comment by JacksonMaxfield** [comment]
@tohuynh had to change `rel="_blank"` to `rel="opener"` because `"_blank"` is a security concern?? https://mathiasbynens.github.io/rel-noopener/

**Comment by tohuynh** [comment]
> @tohuynh had to change `rel="_blank"` to `rel="opener"` because `"_blank"` is a security concern?? https://mathiasbynens.github.io/rel-noopener/

[target=opener](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) is not valid vaue for target

To make it target=_blank safe, you can add rel="noopener noreferrer" (https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types). You can add the "license" rel as well for the license link.

**Comment by JacksonMaxfield** [comment]
> > @tohuynh had to change `rel="_blank"` to `rel="opener"` because `"_blank"` is a security concern?? https://mathiasbynens.github.io/rel-noopener/
> 
> [target=opener](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) is not valid vaue for target
> 
> To make target=_blank safe, you can add rel="noopener noreferrer" (https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types). You can add the "license" rel as well for the license link.

Web stuff is wild. Changing

**Pull request merged**